viam_cartographer.go

```diff
@@ -206,7 +218,9 @@ func New(
 		lidarDataRateMsec:             lidarDataRateMsec,
 		timedLidar:                    timedLidar,
 		imuName:                       imuName,
+		imu:                           imu,
```

Comment:
[q] Do we need both `imu` and `timedIMU`?

Reply:
Yes, since we also need both lidar and timedLidar. It's used as an override in testing, otherwise it is just set to the same value as imu.